edit/edit.js

```diff
@@ -88,27 +120,10 @@ preinit();
   }
 
   function buildThemeElement() {
-    const themeElement = $('#editor.theme');
-    const themeList = localStorage.codeMirrorThemes;
-
-    const optionsFromArray = options => {
-      const fragment = document.createDocumentFragment();
-      options.forEach(opt => fragment.appendChild($create('option', opt)));
-      themeElement.appendChild(fragment);
-    };
-
-    if (themeList) {
-      optionsFromArray(themeList.split(/\s+/));
-    } else {
-      // Chrome is starting up and shows our edit.html, but the background page isn't loaded yet
-      const theme = prefs.get('editor.theme');
-      optionsFromArray([theme === 'default' ? t('defaultTheme') : theme]);
-      getCodeMirrorThemes().then(() => {
-        const themes = (localStorage.codeMirrorThemes || '').split(/\s+/);
-        optionsFromArray(themes);
-        themeElement.selectedIndex = Math.max(0, themes.indexOf(theme));
-      });
-    }
+    CODEMIRROR_THEMES.unshift(chrome.i18n.getMessage('defaultTheme'));
+    $('#editor.theme').append(...CODEMIRROR_THEMES.map(s => $create('option', s)));
```

Comment:
We may want to mention this change in changelog or something that we no longer read codemirror themes from local files.

Reply:
Only if you do it yourself :-) I would wait for a report from a real user who was inventive enough to put custom files there.